src/js/populationSlider.ts

```diff
@@ -143,22 +107,31 @@ const setUpPopulationSlider = (
   sliders.left.setAttribute("max", RANGE_MAX.toString());
   sliders.right.setAttribute("max", RANGE_MAX.toString());
   sliders.right.value = RANGE_MAX.toString();
+  draw(sliders, 0, RANGE_MAX);
 
   const legend = document.querySelector(".population-slider-legend");
-  STRING_INTERVALS.forEach((val) => {
+  POPULATION_INTERVALS.forEach(([intervalText]) => {
     const span = document.createElement("span");
-    span.appendChild(document.createTextNode(val));
+    span.appendChild(document.createTextNode(intervalText));
     legend.appendChild(span);
   });
 
-  draw(sliders, STRING_INTERVALS.at(0), STRING_INTERVALS.at(-1));
+  const onChange = (): void => {
```

Comment:
There's a bug I totally missed in the last refactor. Without using floor(), the right slider moves at 0.5 intervals, which I believe isn't something we want. 
Should I create a separate issue?